docs/advanced-usage.md

```diff
@@ -9,8 +9,11 @@ Various aspects of ORCA's behavior can be altered at runtime through the use of
 These affect ORCA in all contexts.
 
 * <a name="ORCA_COVERAGE_CLOVER"></a>**`ORCA_COVERAGE_CLOVER`**: Change the path where ORCA saves the PHPUnit test coverage Clover XML file.
+* <a name="ORCA_COVERAGE_COBERTURA"></a>**`ORCA_COVERAGE_COBERTURA`**: Change the path where ORCA saves the PHPUnit test coverage Cobertura XML file.
+* <a name="ORCA_COVERAGE_COBERTURA_ENABLE"></a>**`ORCA_COVERAGE_COBERTURA_ENABLE`**: Set to `TRUE` to generate test coverage data in Cobertura format instead of the default Clover format.
 
-* <a name="ORCA_COVERAGE_ENABLE"></a>**`ORCA_COVERAGE_ENABLE`**: Set to `TRUE` to generate test coverage data . The resulting Clover file will be saved at the location set in [`ORCA_COVERAGE_CLOVER`](#ORCA_COVERAGE_CLOVER). Test coverage generation greatly increases build times, so only enable it on one job--all that makes sense anyway.
+* <a name="ORCA_COVERAGE_ENABLE"></a>**`ORCA_COVERAGE_ENABLE`**: Set to `TRUE` to generate test coverage data . By default, the coverage will be generated in Clover format and resulting Clover file will be saved at the location
```

Comment:
I'm afraid we've lost some symmetry here--we have one variable that enables two things while the other enables only one, and one feature that can't be disabled without disabling both. I think what we want is this:

| Enable | Configure |
|---|---|
| `ORCA_COVERAGE_CLOVER_ENABLE` | `ORCA_COVERAGE_CLOVER` |
| `ORCA_COVERAGE_COBERTURA_ENABLE` | `ORCA_COVERAGE_COBERTURA` |
| `ORCA_COVERAGE_ENABLE`* | `ORCA_COVERAGE_CLOVER` |

*For backward compatibility, `ORCA_COVERAGE_ENABLE` should be preserved with the same behavior, thus becoming essentially an alias of the new `ORCA_COVERAGE_CLOVER_ENABLE`. It should _not_ automatically enable Cobertura coverage.